default/scripting/ship_hulls/SH_COLONY_BASE.focs.txt

```diff
@@ -4,9 +4,13 @@ Hull
     speed = 0
     fuel = 0
     stealth = 5
-    structure = 2
-    slots = Slot type = Internal position = (0.50, 0.50)
-    buildCost = 3
+    structure = 3
+    slots = [
+        Slot type = Internal position = (0.38, 0.50)
+        Slot type = Internal position = (0.50, 0.50)
+        Slot type = Internal position = (0.62, 0.50)
+    ]
+    buildCost = 6
```

Comment:
Should this cost be multiplied by upkeep factor? (also ship hull cost multiplier)
buildCost = 6 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_HULL_COST_MULTIPLIER]]
Otherwise, spamming comsats will regain efficiency with larger fleets.

Reply:
done.